src/styles/main.scss

```diff
@@ -1,7 +1,22 @@
 @import "utils";
 @import "fonts";
 @import "typography";
+@import "blocks/header";
+@import "blocks/phone";
+@import "blocks/menu";
+@import "blocks/recommended";
+@import "blocks/categories";
+@import "blocks/buy-reasons";
+@import "blocks/contact";
+@import "blocks/footer";
+@import "blocks/button-wrapper";
+
+html {
+  scroll-behavior: smooth;
+}
 
 body {
-  background: $c-gray;
+  font-weight: 700;
```

Comment:
It's not recommended to set a global font-weight as 700. Use specific font-weight for individual elements or blocks.

Reply:
I moved `font-weight` to elements.